provision/core/mariadb/provision.sh

```diff
@@ -106,6 +106,10 @@ function mysql_setup() {
   cp -f "/srv/provision/core/mariadb/config/my.cnf" "/etc/mysql/my.cnf"
   vvv_info " * Copied /srv/provision/core/mariadb/config/my.cnf               to /etc/mysql/my.cnf"
 
+  cp -f  "/srv/provision/core/mariadb/config/root-my.cnf" "/root/.my.cnf"
```

Comment:
@msaggiorato did you forgot to commit `root-my.cnf`?

Reply:
yes the file is missing :-)

Reply:
It's not a new file, it's the same file copied into `/home/vagrant/.my.cnf`